src/carma-1-tenth/plot_multiple_crosstrack_errors.py

```diff
@@ -0,0 +1,73 @@
+# Plot the crosstrack error as a function of downtrack (distance traveled along the route)
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+import argparse, argcomplete
+import os
+from plot_crosstrack_error import plot_crosstrack_error
+
+
+def plot_multiple_crosstrack_errors(red_point_five_bag, red_point_seven_bag, red_one_bag, blue_point_five_bag, blue_point_seven_bag, blue_one_bag, show_plots=True):
```

Comment:
Is there any easy way you could take an arbitrary number of input bags and add plots for each? That may require some annoying logic so I would alternative suggest a function that generates a single plot from a given bag so we aren't forced to use a specific number of inputs.